index.html

```diff
@@ -33,6 +33,26 @@
         </div>
     </section>
 
+    <section class="wrapper">
+        <div id="stars"></div>
+        <div id="stars2"></div>
+        <div id="stars3"></div>
+        </section>
+
+    <section class="hero">
+        <div id="main">
+            <h1>Mantra Bot</h1>
+        </div>
+        <br>
+        <div id="description">
+            <p>A powerful general purpose bot for your discord server </p>
+        </div>
+        <br>
+        <div id="invite-button">
+            <a href="#"></a><button class="button">Add to Discord</button>
```

Comment:
Would make much more sense to keep `Join Support Server` here as the invite button is already on the navbar.
What do you think?

Reply:
@thestuti